packages/rrweb/test/utils.ts

```diff
@@ -17,7 +17,7 @@ import * as fs from 'fs';
 
 export async function launchPuppeteer() {
   return await puppeteer.launch({
-    headless: process.env.PUPPETEER_HEADLESS ? true : false,
+    headless: process.env.PUPPETEER_DEBUG ? false : true,
```

Comment:
I switched this to default to headless, as actually I got some flaky tests locally without this. IMHO headless makes sense as default, unless you really want to  debug the chrome instance. If you think that's a bad change, I can revert this!

Reply:
for the flaky local tests, I've had better success with `runInBand`